sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java

```diff
@@ -71,6 +66,46 @@ public void useAadAsyncClient() {
             .buildAsyncClient();
     }
 
+    /**
+     * Code snippet for rotating API key of the client
+     */
+    public void rotatingApiKey() {
+        TextAnalyticsApiKeyCredential credential = new TextAnalyticsApiKeyCredential("{api_key}");
+        TextAnalyticsClient textAnalyticsClient = new TextAnalyticsClientBuilder()
+            .apiKey(credential)
+            .endpoint("{endpoint}")
+            .buildClient();
+
+        credential.updateCredential("{new_api_key}");
+    }
+
+    /**
+     * Code snippet for handling exception
+     */
+    public void handlingException() {
+        List<DetectLanguageInput> inputs = Arrays.asList(
+            new DetectLanguageInput("1", "This is written in English.", "us"),
+            new DetectLanguageInput("1", "Este es un document escrito en Español.", "es")
+        );
+
+        try {
+            textAnalyticsClient.detectLanguageBatch(inputs, null, Context.NONE);
```

Comment:
if the method takes the Context parameter, shouldn't it be `withResponse`?

Reply:
There will be no "withResponse" for pagination. See the pagination example: https://azure.github.io/azure-sdk/java_design.html#pagination